osclib/accept_command.py

```diff
@@ -78,11 +128,15 @@ def perform(self, project, force=False):
             oldspecs = self.api.get_filelist_for_package(pkgname=req['package'],
                                                          project=self.api.project,
                                                          extension='spec')
-            change_request_state(self.api.apiurl,
-                                 str(req['id']),
-                                 'accepted',
-                                 message='Accept to %s' % self.api.project)
-            self.create_new_links(self.api.project, req['package'], oldspecs)
+            if 'type' in req and req['type'] == 'delete' and self.api.delreq_monitor:
+                # Virtual accept to the delete request
+                self.virtual_accept_to_delete(req['id'], req['package'])
```

Comment:
nit: just as above the `to` is a bit odd. Perhaps `virtual_accept_delete` or `virtually_accept_delete`.